### ISSUE TITLE
Fix pkgdown configuration for reference

### ISSUE THREAD

**Issue by maelle** [ropensci/NLMR#100]
Some topics are missing from the configuration file.

```r
-- Building function reference -------------------------------------------------
Error in check_missing_topics(rows, pkg) : 
  All topics must be included in reference index
• Missing topics: NLMR-package
```

Note that for topics you do not want to include in the index you can create an "internal" section https://pkgdown.r-lib.org/reference/build_reference.html?q=internal#missing-topics

To check all topics are listed, after editing the configuration file you can run `pkgdown::check_pkgdown()`.

**Comment by maelle** [comment]
@marcosci @bitbacchus (in case you missed this, not that urgent :sweat_smile: )

**Comment by mhesselbarth** [comment]
Should be fixed in #100 - please check and merge
